PREREQUISITES.md

```diff
@@ -45,7 +45,7 @@ activation script (e.g. `source /opt/intel/sgxsdk/environment`)
 - If you build your own OpenSSL (not the usual case),
   `PKG_CONFIG_PATH` and `LD_LIBRARY_PATH` also contain the the path to
   [OpenSSL](#openssl) package config files and libraries, respectively.
-  You need to do this when pre-built OpenSSL version 1.1.1d or later
+  You need to do this when pre-built OpenSSL version 1.1.1f or later
```

Comment:
Why are you changing openssl version version from 1.1.1d to 1.1.1f?

Reply:
When compiling Python module in standalone build is reporting that the ssl module failed to compile due to missing flags despite it existing for version OpenSSL 1.1.1d. No such error occurred in Openssl1.1.1f for standalone build.